tests/cypress/src/DashTable.ts

```diff
@@ -15,7 +15,17 @@ export default class DashTable {
         return cy.get(`#table tbody tr td.dash-select-cell`).eq(row);
     }
 
+    static getActiveCell() {
+        return cy.get(`#table tbody td.focused`);
+    }
+
     static getSelectedCells() {
         return cy.get(`#table tbody td.cell--selected`);
     }
+
+    static toggleScroll(toggled: boolean) {
+        cy.get('.row-1').then($el => {
+            $el[0].style.overflow = toggled ? '' : 'unset';
+        });
+    }
```

Comment:
Cypress auto-scrolls very aggressively to guarantee element visibility -- this causes issues with the table's own scrolling mechanism and event hooks -- this allows disabling/enabling scrolling on the content section at will.